text/0000-add-contributer-graph.md

```diff
@@ -0,0 +1,71 @@
+# Meta
+[meta]: #meta
+- Name: Add Contributer Graph
+- Start Date: 2020-03-08
+- Author(s): aemengo
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This RFC proposes adding a customized contributer graph to the buildpacks [website](https://buildpacks.io/community/). Please see [mockup](#prior-art) for concept.
+
+# Definitions
+[definitions]: #definitions
```

Comment:
I think it could be helpful to define `health` of an `OSS community` as well, and clarify why it is important to be conscious of, both for the maintainers, as well as for organizations/individuals who are thinking of adopting it.